functions/transformations/range.go

```diff
@@ -140,12 +139,12 @@ func createRangeTransformation(id execute.DatasetID, mode execute.AccumulationMo
 	cache := execute.NewTableBuilderCache(a.Allocator())
 	d := execute.NewDataset(id, mode, cache)
 
-	bounds := a.StreamContext().Bounds()
-	if bounds == nil {
-		return nil, nil, errors.New("nil bounds supplied to range")
+	bounds := execute.Bounds{
```

Comment:
I guess we'll need to do something similar in the `platform` repo too, in `createFromSource`.